src/features/dialog-manager/actions/error-message.js

```diff
@@ -0,0 +1,8 @@
+export default function errorMessage(errorMessage) {
+    return dispatch => {
+        dispatch({
+            type: 'ERROR_MESSAGE',
+            payload: { name: errorMessage, item: errorMessage },
```

Comment:
I don't think it makes sense to pass the error message in as an item, considering it's not an item (i.e. Old Ring, Health Potion, Steel Sword).